locales/en.json

```diff
@@ -263,6 +264,92 @@
+      },
+      "graphs": {
+        "title": "Graphs",
+        "description": "rotki premium gives you access to an array of analytics and graphs. You can see, among others, the analysis of total net value over time, amount and value per asset over time, distribution of your total worth per asset or per location."
+      },
+      "defi_protocol": {
+        "title": "Information per DeFi protocol",
+        "description": "You can see extra information per supported DeFi protocols. For example analysis of your liquity, aave, compound, makerdao positions. Analysis of your yearn vaults. Breakdown of the swaps made in uniswap, sushiswap, balancer and more."
+      },
+      "data_synchronization": {
+        "title": "Data synchronization",
+        "description": "By being a premium user you have the option to upload your account's encrypted database to our server. It can then be synchronized between different devices. It also serves as a backup. The data is encrypted with your password and we can not read it. Synced databases are automatically deleted a month after your subscription expires."
+      },
+      "dark_mode": {
+        "title": "Dark Mode",
+        "description": "Dark mode can lessen the strain on your eyes, improve the readability of text and prolong the battery life of your device."
+      },
+      "monerium_api_keys": {
+        "title": "Monerium API Keys",
+        "description": "Once you provide Monerium credentials as a premium user, all Monerium transactions on the mainnet, Polygon, and Gnosis chain will be enriched with bank data or chain bridging information. You can also view this data in the Monerium app and have SEPA data overlaid on your transactions."
+      },
+      "theme_customization": {
+        "title": "Theme customization",
+        "description": "Premium users have the option to personalize the color scheme for both light and dark modes. Users can access and customize specific color palettes for the app theme within the settings."
+      },
+      "timeframe": {
+        "title": "Customization of timeframe selector on the dashboard",
+        "description": "Set the default time frame for the dashboard graph. This timeframe will be pre-selected upon login. By default, it will remember the previous session’s selection. Premium users will have access to all options, while free users can only choose from \"Remember\", \"2 Weeks\", and \"1 Week\" in the settings."
+      },
+      "limits": {
+        "title": "Increased event limits",
+        "transactions": {
+          "title": "Ethereum transactions",
+          "description": "Free users have a limit of the 100 most recent ethereum transactions in the transactions viewer. As a premium user you have higher limits in the amount of transactions you can see."
+        },
+        "trades": {
+          "title": "Trades",
```

Comment:
I would say combine this with the history evrents view. And just use "limits" and higher limits. Do not specify. Same for deposits/wtihdrawals.

Instead if you want another section add the notes. Free users can only add up to 10 notes.